### ISSUE TITLE
not able to add this package V1.1.1

### ISSUE THREAD

**Issue by akramercury** [uvdesk/ecommerce#10]
I am not able to add this package I think there is something wrong with the how the 

php bin/console uvdesk_extensions:configure-extensions

structure the files

**Comment by DmitryVL** [comment]
I couldn't install this package either. There are no errors when installing, but nothing is added when finished.

**Comment by DmitryVL** [comment]
When I try to install the app I get the message:
Invalid package extension.json file. The qualified package name should be '  
  ecommerce/Controller' but the specified name is 'uvdesk/ecommerce'

**Comment by akramercury** [comment]
Yes same here, I think the package is not properly mapped trying to figure out how to fix this

**Comment by komal-sh-27** [comment]
@akramercury , @DmitryVL 

Let us know, you have first create a **uvdesk folder in apps folder**: **your-project-folder/apps/uvdesk** after that clone this package to the **your-project-folder/apps/uvdesk/ecommerce** directory relative to your project's root. 

Once you've copied the package to the specified directory, run the following command in your project's root:

```
$ php bin/console uvdesk_extensions:configure-extensions
$ php bin/console assets:install
$ php bin/console doctrine:migrations:diff
$ php bin/console doctrine:migrations:migrate
```

Once you check the below link of installation video and once try with the same steps. 

https://www.awesomescreenshot.com/video/14014921?key=47a687cfd5ab26a22f13b4d4519a7e89 

Also, let us know you are facing the same error.
